metricbeat/mb/testing/data/README.md

```diff
@@ -26,7 +26,8 @@ An alternative is to just run from metricbeat `mage mockedTests` to achieve the
 ### Available settings in `config.yml`
 
 - `path`: (string) Path to reach the directory containing the files to read from. Default is "_meta/testdata".
-- `wirtepath`: (string) Path to the directory where the expected files are going to be written. Default is "_meta/testdata".
+- `writepath`: (string) Path to the directory where the expected files are going to be written. Default is "_meta/testdata".
+- `datapath`: (string) Path to the directory where the data.json file is going to be written. Default is "_meta".
```

Comment:
could you please somewhere add a comment why we have `data.json` file and that is used/might be used for our public documentation (or link to the place where it is mentioned, but I couldn't find it)

Reply:
I can try, but I am not sure I understand the reasoning for that file since it was already there

Reply:
I was thinking about adding smth like `data.json file is used to render example events published in our official documentation`
at least for me it wasn't clear at the moment of asking why we generated those files in this comment https://github.com/elastic/beats/pull/34467#discussion_r1101788362

Reply:
Ok, done that! I was thinking something more complex, but as you wrote it, it makes perfect sense. I don't want to link to the line code because I would have to use one fixed commit and that might end up not making sense if we are on the branch main and are sent to a possible very old one.